code-challenges.test.js

```diff
@@ -32,12 +32,18 @@ describe("fibFunction", () => {
 
 // Create a function that takes a number > 2 as an argument.
 // Using Fibonacci sequence - adding the 2 previous numbers together for the new number
-// Using a for loop, have the indexes of number that is less than the input number returned in a new variable that stores them.
-// Create another function that uses the new variable to add the two previous indexes together and create another array that stores these values.
-// Use .map() method to return the value[i] of the number input
-
-  fibFunction = (numberInput) => {
+// Create array that stores the first 2 indexes of the fib sequence [1, 1]
+// Use a for loop that adds one more index until it is 2 less than the input number.
+// Return needs to be outside of for loop function, otherwise when it was inside the for loop, it quit iterating at 2 indexes added together [1,1]
+//
 
+// declare the variable and set it to a function
+const fibFunction = (number) => {
+  let fibArray = [1, 1]
+  for(let i = 0; i < number - 2; i ++) {
+    fibArray.push(fibArray[i] + fibArray[i + 1])
+  }
+  return fibArray
```

Comment:
Much better approach! If you want to level your skill up come back through here and write down all of the things happening in this code in plain language!